src/utilities/_hypre_utilities.hpp

```diff
@@ -602,14 +602,6 @@ using hypre_DeviceItem = sycl::nd_item<3>;
       hypre_assert(0); exit(1);                                                              \
    } } while(0)
 
-#define HYPRE_CUSOLVER_CALL(call) do {                                                       \
```

Comment:
What's this change about @victorapm ?

Reply:
We had this defined twice, so I removed the redundant one

Reply:
Got it.  Thanks!